meshmode/mesh/processing.py

```diff
@@ -62,6 +62,367 @@ def find_group_indices(groups, meshwide_elems):
 
 # {{{ partition_mesh
 
+def _create_element_part_map(part_per_element, parts, element_id_dtype):
```

Comment:
```suggestion
def _compute_global_elem_to_part_elem(part_per_element, parts, element_id_dtype):
```
(for recognizability below)